R/extract.R

```diff
@@ -293,18 +293,26 @@ extract.betamfx <- function(model, include.pseudors = TRUE,
   return(tr)
 }
 
-#' \code{\link{extract}} method for \code{betamfx} objects
+#' \code{\link{extract}} methods for \code{mfx.objects}, statistical models 
```

Comment:
I'm putting together two methods from the same package. However, the Rd file prints only the name of the first method. As you can see here I tried to trick it by adding a collective name for the two. Of course, it didn't work because the method I invented does not exist. In the `lme4` case it worked smoothly because it is also the name of the package. However, when none of the functions documented matches with the name of the package I think this gets quite confusing for the user that cannot find documentation for the method of interest, but who should know the other functions set as methods from the same package in order to find it. For this reason, I would not do it this way unless the name of the package matches the method. What do you think?

Reply:
The title and description should be general enough to describe both methods. For example, you could write "Methods for extracting data from models in the mfx package" in the title. The description already looks good. The `@rdname` tag bundles the two methods together. You can use the name of the first method for this. To make sure that the user can find this page from the perspective of either function, you can use the `@aliases` field. Alternatively, if you think the file name should be reflective of the package name, it's also possible to first create a dummy entry with `NULL` contents that describes it in a general way and then use minimal Roxygen markup for the two functions. Take a look at Hadley's documentation for `@rdname`. He has a page somewhere on documenting multiple functions together, and there is an example on creating a dummy entry first. Let me know if you don't find it. I'm on my phone now but can look it up tomorrow if you need it.

Reply:
Ok! Thanks, I'll look it up and ask you again in case I can't find it!